server/reviewer/queries.ts

```diff
@@ -0,0 +1,12 @@
+import { Reviewer } from 'server/models';
+
+export const createReviewer = (inputValues) => {
+	return Reviewer.create({
```

Comment:
I think you can write:

```ts
return Reviewer.create({ ... }, { returning: true });
```

 to get Sequelize to return the created model so you don't have to re-query it (it's a little odd that they don't do that by default.)